### PR TITLE
chore: Updated package version of json5 and @babel/core

### PR DESCRIPTION
**Pull request by sauravlikhar**
Updating the package json5 to fix the alerts

### CODE REVIEW

**Comment by pavelfeldman** [comment]
Bots don't look happy, you should run `node ./utils/generate_third_party_notice.js`.

File: packages/playwright-test/bundles/utils/package-lock.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "name": "utils-bundle",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```
Comment (mxschmitt) [inline packages/playwright-test/bundles/utils/package-lock.json:4]: this is not what we want I think, lets stay on lockfileversion 2 for now?

File: packages/playwright-test/bundles/babel/package-lock.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "babel-bundle",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```
Comment (mxschmitt) [inline packages/playwright-test/bundles/babel/package-lock.json:4]: ditto

File: packages/playwright-test/bundles/utils/package-lock.json (hunk 1 above)
Comment (sauravlikhar) [reply packages/playwright-test/bundles/utils/package-lock.json]: Sure, reverted the lockfile version.

File: packages/playwright-test/bundles/babel/package-lock.json (hunk 2 above)
Comment (sauravlikhar) [reply packages/playwright-test/bundles/babel/package-lock.json]: Done

File: packages/playwright-test/ThirdPartyNotices.txt [hunk 3]
```diff
@@ -6,39 +6,39 @@ This project incorporates components from the projects listed below. The origina
 
 -	@ampproject/remapping@2.2.0 (https://github.com/ampproject/remapping)
 -	@babel/code-frame@7.18.6 (https://github.com/babel/babel)
--	@babel/compat-data@7.20.1 (https://github.com/babel/babel)
--	@babel/core@7.20.2 (https://github.com/babel/babel)
--	@babel/generator@7.20.4 (https://github.com/babel/babel)
+-	@babel/compat-data@7.20.14 (https://github.com/babel/babel)
```
Comment (pavelfeldman) [inline packages/playwright-test/ThirdPartyNotices.txt:9]: There should be no changes other than the json5.

File: packages/playwright-test/ThirdPartyNotices.txt [hunk 4]
```diff
@@ -114,17 +114,18 @@ This project incorporates components from the projects listed below. The origina
 -	jest-regex-util@27.5.1 (https://github.com/facebook/jest)
 -	js-tokens@4.0.0 (https://github.com/lydell/js-tokens)
 -	jsesc@2.5.2 (https://github.com/mathiasbynens/jsesc)
--	json5@2.2.1 (https://github.com/json5/json5)
+-	json5@2.2.2 (https://github.com/json5/json5)
```
Comment (pavelfeldman) [inline packages/playwright-test/ThirdPartyNotices.txt:117]: should be -2.2.1 +2.2.3

File: packages/playwright-test/bundles/utils/package.json [hunk 5]
```diff
@@ -9,7 +9,7 @@
     "generate-license": "node ../../../../utils/generate_third_party_notice.js"
   },
   "dependencies": {
-    "json5": "2.2.1",
+    "json5": "2.2.2",
```
Comment (pavelfeldman) [inline packages/playwright-test/bundles/utils/package.json:12]: Did you mean 2.2.3?

File: packages/playwright-test/ThirdPartyNotices.txt (hunk 3 above)
Comment (sauravlikhar) [reply packages/playwright-test/ThirdPartyNotices.txt:9]: In playwright-core/packages/bundles, json5 is not present in package.json but it is coming as dependency of @babel/core. So on updating @babel/core only we can get json5 updated.

File: packages/playwright-test/ThirdPartyNotices.txt (hunk 4 above)
Comment (sauravlikhar) [reply packages/playwright-test/ThirdPartyNotices.txt]: Updated

File: packages/playwright-test/bundles/utils/package.json (hunk 5 above)
Comment (sauravlikhar) [reply packages/playwright-test/bundles/utils/package.json]: Yes, updated the file.

**Review by mahtabnejad90 — APPROVED** [review]

File: packages/playwright-test/bundles/babel/package.json [hunk 6]
```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@babel/code-frame": "^7.18.6",
-    "@babel/core": "^7.20.2",
+    "@babel/core": "^7.20.12",
```
Comment (pavelfeldman) [inline packages/playwright-test/bundles/babel/package.json:13]: Now that you are updating babel/core, you should keep this dependency update consistent across babel packages below as much as possible.

Comment (sauravlikhar) [reply packages/playwright-test/bundles/babel/package.json:16]: Sure, updated all the babel packages to make them consistent.

**Pull request closed without merging**